karrot/groups/tests/test_model.py

```diff
@@ -81,3 +82,50 @@ def test_does_not_count_from_other_groups(self):
         )
         memberships = self.group.groupmembership_set.pickup_active_within(days=7)
         self.assertEqual(memberships.count(), 0)
+
+
+class TestGroupManager(TestCase):
+
+    # test if setting a default status and theme via settings works
+
+    # set each setting individually
+    @override_settings(GROUP_THEME_DEFAULT=themes.GroupTheme.FOODSAVING)
+    def test_create_group_theme_default_foodsaving(self):
+        this_object = Group.objects.create()
```

Comment:
Looks good!
Maybe write `group = GroupFactory()` for consistency with other tests (have look at `TestGroupModel` in the same file).